index.rst

```diff
@@ -131,8 +131,8 @@ special characters for wildcard matching:
 The backslash character (``\\``) can be used to escape a character so it is
 not interpreted as a special character.
 
-The maximum length of a section name is 4096 characters. All sections
-exceeding this limit are ignored.
+The minimum supported section name length is 1024 characters.
+Beyond that, each implementation can define its own upper limit.
```

Comment:
```suggestion
Beyond that, each implementation may choose to define its own upper limit or no explicit limit at all.
```

Reply:
Oh yeah thanks a lot for this! <3 (and the other suggestion)